pkg/release/release.go

```diff
@@ -195,7 +193,14 @@ func (r *Release) Sync(client helm.Client, hr *v1.HelmRelease) (rHr *v1.HelmRele
 	// technically return an error.
 	v, _ := composedValues.YAML()
 
-	var performRollback bool
+	var (
+		performRollback bool
+		maxHistory      int = 10
```

Comment:
Instead of setting this here I would simply move the default to the `HelmRelease` itself, an example of how this is done [can be found here](https://github.com/fluxcd/helm-operator/blob/master/pkg/apis/helm.fluxcd.io/v1/types.go#L217).

Reply:
Done!